src/google/protobuf/arena.h

```diff
@@ -467,7 +467,7 @@ class PROTOBUF_EXPORT PROTOBUF_ALIGNAS(8) Arena final {
     }
 
     template <typename U>
-    static Arena* GetArenaForAllocation(Rank2, const U* p) {
+    static Arena* GetArenaForAllocation(Rank2, const U*) {
```

Comment:
inside google these are used.  Please switch the suppression technique to

`(void)p;`